integration/repo_test.go

```diff
@@ -16,6 +16,7 @@ import (
 
 // TestRepository tests `trivy repo` with the local code repositories
 func TestRepository(t *testing.T) {
+	t.Setenv("NUGET_PACKAGES", "/tmp/fake_nuget_packages")
```

Comment:
What if using [t.TempDir](https://pkg.go.dev/testing#T.TempDir)?

```suggestion
	t.Setenv("NUGET_PACKAGES", t.TempDir())
```

Reply:
Makes sense. Fixed https://github.com/aquasecurity/trivy/pull/8107/commits/26fb4f93f6d09dd17f854830a47c53d2b6dbef85